### PR TITLE
Chore:  get single comment

### PR DESCRIPTION
**Pull request by neymarjimoh**
**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**
-  This PR contains test suite that will cover all the functionalities expected from the getSingleComment endpoint

Fixes #issue_number (replace issue_number with the issue number this PR solves)
#339 

Summarise the main tasks handled in the pull request

**description of Task to be completed?**

Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**
- checkout to this branch `git checkout chore-getSingle-comment`
- run `npm install`
- run `npm run test`

Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)

**Any background context you want to provide?**
- None
Any pertinent information that should be considered
- None
**What is the link to the issue on Github?**

[Issue #339 (https://github.com/microapidev/comment-microapi/issues/339)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here

### CODE REVIEW

**Comment by simply-alliv** [comment]
Firstly, your build is failing. We can't merge any work that has a failing build.

Secondly, you are not testing all parts of the implementation of the endpoint. What about tests for the 422 errors?

This is an integration test, as it is testing multiple dependencies within a functionality, for the client API so please be more rigorous with your testing as we do not want bugs creeping into the public-facing API.

File: __test__/api/comments/getSingleComment.test.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  it("should 404 error for inavlid comment Id", async () => {
+    const bearerToken = `bearer ${global.appToken}`;
+    const commentId = "12345Avd"; // invalid ID
+
+    const res = await request
+      .get(`/v1/comments/${commentId}`)
+      .set("Authorization", bearerToken);
+
+    expect(res.status).toBe(404);
+    expect(res.body.message).toEqual("invalid ID");
```
Comment (piouson) [inline __test__/api/comments/getSingleComment.test.js:75]: run test locally and use correct response..

```bash
npm run test __test__/api/comments/getSingleComment.test.js
```

Comment (neymarjimoh) [reply __test__/api/comments/getSingleComment.test.js:75]: Ok.. thanks

**Comment by neymarjimoh** [comment]
I am querying the db to get some data from the API, how can I test for 422 error @simply-alliv 
I'm not entering anything on the body

**Pull request merged**